CaPPMS/Shared/FaqQuestion.razor

```diff
@@ -0,0 +1,43 @@
+﻿@using CaPPMS.Model
+@namespace CaPPMS.Shared
+
+<div class="panel panel-default border @CollapseQuestion">
+    <div class="panel-body">
+        <div class="container-fluid">
+            <div class="row">
+                <div>
+                    <button @onclick="e => this.IsCollapsedReply = !this.IsCollapsedReply">
+                        @(this.IsCollapsedReply ? "+" : "-")
+                    </button>
+                </div>
+                <div class="font-italic" style="text-align:left">&nbsp;@faqInfo.Question</div>
+            </div>
+            <FaqReply
+                CollapseReply="@(this.IsCollapsedReply ? "collapse" : "")"
+                FaqInformation="@faqInfo">
+
+            </FaqReply>
+        </div>
+    </div>
+</div>
+
+@code {
+    [Parameter]
+    public string CollapseQuestion { get; set; } = "collapse";
```

Comment:
Does this need to be public?

Reply:
When I make this private, the editor yells at me, so I made it public so it would stop